frontend/src/static/js/components/webstatus-feature-page.ts

```diff
@@ -143,6 +143,33 @@ export class FeaturePage extends LitElement {
           gap: var(--content-padding-large);
         }
 
+        sl-card .card {
+          height: 100%;
+        }
+
+        .wptScore {
+          width: 16%;
```

Comment:
This is fine, but if we ever come back to this, maybe we should try grid-layout with fractions.

Reply:
+1

@dlaliberte,

Could you create a follow up issue for this ?

Reply:
Sure, I'l like to figure out grid CSS enough to implement this layout.  If it avoids the arbitrary percentages, that would be a plus.

Reply:
Added #331 to address this.